### PR TITLE
add abbr for artifact and model registry

### PR DESCRIPTION
**Pull request by dberenbaum**
Adds `<abbr>` with tooltips for two new basic concepts: `artifact` and `model registry`.

### CODE REVIEW

File: content/docs/dvclive/live/log_artifact.md [hunk 1]
```diff
@@ -37,20 +37,18 @@ with Live() as live:
 ## Description
 
 Log `path`, saving its contents to DVC storage. Also annotate with any included
-metadata fields (for example, to be consumed in [Studio model registry] or
-automation scenarios).
+metadata fields (for example, to be consumed in the <abbr>model registry</abbr>
+or automation scenarios).
```
Comment (dberenbaum) [inline content/docs/dvclive/live/log_artifact.md:41]: Tried to be consistent with how we refer to model registry throughout the docs

File: content/docs/gto/command-reference/show.md [hunk 2]
```diff
@@ -31,12 +31,6 @@ $ gto show -r https://github.com/iterative/example-gto
 ╘══════════╧══════════╧════════╧═════════╧════════════╛
 ```
 
-Here we'll see artifacts that have Git tags or are annotated in
-`artifacts.yaml`. The artifacts that have annotation, but have no Git tags, are
-considered yet `unregistered` and will be marked with an asterisk, e.g.
-`*annotated`. Use `--all-branches` or `--all-commits` to read `artifacts.yaml`
-from more commits than just `HEAD`.
```
Comment (dberenbaum) [inline content/docs/gto/command-reference/show.md:38]: This looks to be deprecated AFAICT

File: content/basic-concepts/artifact.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+name: Artifact
+match: [artifact, artifacts]
+tooltip: >-
+  An artifact is a model or other file or directory for which structured
```
Comment (shcheklein) [inline content/basic-concepts/artifact.md:5]: minor / style: can we restyle it please? keep it under 80 symbols per line

File: content/basic-concepts/model_registry.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+name: Model Registry
+match: [model registry]
+tooltip: >-
+  The model registry connects all of your team's models through Git.  Find, organize,
```
Comment (shcheklein) [inline content/basic-concepts/model_registry.md:5]: minor: extra space could be removed before `Find ...`

**Review by shcheklein — APPROVED** [review]

**Comment by omesser** [comment]
pushed formatting fix in [de786e8](https://github.com/iterative/dvc.org/pull/4844/commits/de786e8cd9af2cbd9f26a7c932d80468aa5e3121)

**Pull request merged**